src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,396 @@
+
+## Microscopy recording data
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+Template:
+```Text
+sub-<label>/
+    [ses-<label>/]
+        microscopy/
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.<extension>
+            sub-<label>[_ses-<label>]_sample-<label>[_acq-<label>][_stain-<label>][_run-<index>][_chunk-<index>]_<suffix>.json
+```
+Microscopy data MUST be stored in the `microscopy` directory.
+
+### File formats
+The Microscopy community uses a variety of formats for storing raw data, and there is no single
+standard that all researchers agree on. However, a standardized file structure has been developed
+by the [Open Microscopy Environment](https://www.openmicroscopy.org/) for whole-slide imaging with
+the [OME-TIFF file specifications](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/file-structure.html).
+The OME-TIFF file allows for multi-page TIFF files to store multiple image planes and supports
+multi-resolution pyramidal tiled images. An OME-XML data block is also embedded inside the
+file’s header.
+
+The BIDS standard accepts microscopy data in different file formats to accommodate datasets
+stored in 2D image formats and whole-slide imaging formats, to accommodate lossless and lossy
+compression, and to avoid unnecessary conversions of the original data from a non-tiled to a
+tiled format, or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats.
+For example with the [OME-NGFF](https://ngff.openmicroscopy.org/latest/) format currently
+developed by OME as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Name**                                      | `suffix` |
+| --------------------------------------------- | -------- |
+| Transmission electron microscopy              | TEM      |
```

Comment:
Nice! What is the source of this vocabulary? This might be some low-hanging fruit for coordinating nomenclature.

Reply:
Thank you!
This list was created by the BEP contributors, we started with a few modalities and then added some and/or changed the names/suffixes based on comments from the community on the most common modalities/abbreviations used.